Typo cleanup for app

akiojalehto opened the pull request:
### Description
I was adjusting my code inspection and made a little larger cleanup for typos while at it. No breaking changes, just misspellings.

### Contribution checklist
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [X] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

akiojalehto committed:
Fix typo in comments

akiojalehto committed:
Fix typo in DocBlock comment

akiojalehto committed:
Fix typos in DocBlock comment

akiojalehto committed:
Fix typos in DocBlock comment

akiojalehto committed:
Fix typos in array keys of meta data

akiojalehto committed:
Fix typos in DocBlock comment

akiojalehto committed:
Fix typos in DocBlock comment

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typos in DocBlock comment

akiojalehto committed:
Fix typo in property name

akiojalehto committed:
Fix typos in DocBlock comment

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typos in method names

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typo in method name#

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typos in comments

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typo in property name

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typos in variable name

akiojalehto committed:
Fix typos in method names

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typo in method names

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typo in method names

akiojalehto committed:
Fix typos in DocBlock comments

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typos in comments

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typo in method names

akiojalehto committed:
Fix typos in comments

akiojalehto committed:
Fix typo in method names

akiojalehto committed:
Fix typos in comments

akiojalehto committed:
Fix typo in property name

akiojalehto committed:
Fix typos in comments

akiojalehto committed:
Fix typo in property name

akiojalehto committed:
Fix typos in comments

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typos in comments

akiojalehto committed:
Fix typo in method names

akiojalehto committed:
Fix typos in comments

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typo in comments

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typo in comments

akiojalehto committed:
Fix typo in property name

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typo in property name

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typo in comments

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typo in comments

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typo in variable name

akiojalehto committed:
Fix typo in method name

akiojalehto committed:
Fix typo in property name

akiojalehto committed:
Revert some changes to pass unit tests?

akiojalehto committed:
Merge remote-tracking branch 'origin/2.3-develop' into fix/typos/batch

ishakhsuvarov reviewed (approved):
Changes look good to me. I am going to invite more people for the review just in case, as change set is rather big.

ishakhsuvarov committed:
Suppress warnings on an unused constructor parameter

 - Suppressed warning for unused formal parameter in constructor so that static tests pass
 - Marked unused property as deprecated for future removal

ishakhsuvarov commented:
@akiojalehto Thank you for contribution. Changes look good to me.
I had to add a suppress warning in one place as tests detect issues not related to this PR (Static tests are executed only on the files changed).

akiojalehto commented:
@ishakhsuvarov Great, thank you. I noticed that the test result was unrelated to PR, but didn't spot the logic for it.

orlangur commented on app/code/Magento/Catalog/Api/Data/ProductRender/PriceInfoInterface.php:
```diff
@@ -51,7 +51,7 @@ public function setMaxPrice($maxPrice);
     /**
      * Set max regular price
-     * Max regular price is the same, as maximum price, except of excluding calculating special price and catalogules
+     * Max regular price is the same, as maximum price, except of excluding calculating special price and catalogues
```
catalog rules maybe?

ishakhsuvarov replied:
Looks like yes

ishakhsuvarov committed:
Update phpDoc with more appropriate description

ishakhsuvarov commented:
@akiojalehto I took the liberty of updating the doc block with more appropriate wording. Hope you don't mind

akiojalehto commented:
@ishakhsuvarov Don't mind at all, it's much faster if you guys update directly all the things you see more appropriate.
